includes/class-hooks.php

```diff
@@ -225,10 +225,20 @@ public static function configure_theme() {
 
 		register_nav_menus( $menus );
 
+		$html5_features = [
+			'comment-list',
+			'comment-form',
+			'search-form',
+			'gallery',
+			'caption',
+			'style',
+			'script',
+		];
+
+		add_theme_support( 'html5', $html5_features );
```

Comment:
```suggestion
		add_theme_support( 'html5', [
			'comment-list',
			'comment-form',
			'search-form',
			'gallery',
			'caption',
			'style',
			'script',
		] );
```

Reply:
Since we're only using this variable once, we can insert the array directly.

Reply:
Tried this, but [WPCS doesn't like multi-line arguments](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#multiline-function-calls):
> Each parameter must take up no more than a single line. Multi-line parameter values must be assigned to a variable and then that variable should be passed to the function call.

Reply:
Weird!